packages/core/src/modules/credentials/protocol/v2/__tests__/v2-credentials.e2e.test.ts

```diff
@@ -215,7 +215,7 @@ describe('v2 credentials', () => {
     })
   })
 
-  test('Faber issues credential which is then deleted from Alice`s wallet', async () => {
+  xtest('Faber issues credential which is then deleted from Alice`s wallet', async () => {
```

Comment:
Why are these disabled?

Reply:
Oops. Disabled during testing, must have forgot to turn them back on